plugins/webhooks/github.js

```diff
@@ -38,6 +38,7 @@ function pullRequestOpened(options, request, reply) {
     const buildFactory = request.server.app.buildFactory;
     const eventId = options.eventId;
     const pipelineId = options.pipelineId;
+    const jobId = options.jobId;
```

Comment:
I'm not clear how the jobId is passed here when the job hasn't been created yet.